src/java.base/share/classes/java/util/Random.java

```diff
@@ -118,17 +125,18 @@ private static long seedUniquifier() {
     }
 
     private static final AtomicLong seedUniquifier
-        = new AtomicLong(8682522807148012L);
+            = new AtomicLong(8682522807148012L);
 
     /**
      * Creates a new random number generator using a single {@code long} seed.
      * The seed is the initial value of the internal state of the pseudorandom
      * number generator which is maintained by method {@link #next}.
      *
-     * <p>The invocation {@code new Random(seed)} is equivalent to:
-     *  <pre> {@code
-     * Random rnd = new Random();
-     * rnd.setSeed(seed);}</pre>
+     * @implSpec <p>The invocation {@code new Random(seed)} is equivalent to:
```

Comment:
This is not conventional formatting for an implSpec section. I suggest putting implSpec on its own line and then left-justifying the text as normal. A new paragraph tag is no needed at the beginning of implSpec.

Reply:
Adjusted